easybuild/easyconfigs/c/Critic2/Critic2-1.1dev-foss-2023a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'Critic2'
+version = '1.1dev'
+
+homepage = 'https://aoterodelaroza.github.io/critic2/'
+description = """Critic2 is a program for the analysis of quantum mechanical
+calculation results in molecules and periodic solids."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'extra_fflags': '-ffree-line-length-none'}
+
+source_urls = ['https://github.com/aoterodelaroza/critic2/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['47c5b56ec370ae8f46b2248e8d0ed70b9d5b736b04ffe3936de5039d056ca785']
+
+configopts = '-DLIBCINT_INCLUDE_DIRS=$EBROOTLIBCINT/include/ '
+configopts += '-DLIBCINT_LIBRARY=$EBROOTLIBCINT/lib64/libcint.so '
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+]
+
+dependencies = [
+    ('libxc', '4.3.4'),  # use xc_f90_types_m and libxc_funcs_m need libxc < 5.0
+    ('libcint', '5.4.0'),
+    ('libreadline', '8.2'),
+]
+
+sanity_check_paths = {
+    'files': ["bin/critic2"],
+    'dirs': ["bin"],
+}
+
```

Comment:
Can you add `sanity_check_commands`?

For example 
`citric2 -h`

Reply:
Added in the new PR :)